### PR TITLE
Include required metadata in release artifacts and bump version

### PR DESCRIPTION
**Pull request by ben-m-lucas**
## Describe your changes

This PR:
* includes the required metadata for our release artifacts and bumps the version

### CODE REVIEW

File: sdk/pom.xml [hunk 1]
```diff
@@ -5,12 +5,33 @@
     <parent>
         <groupId>com.hiddenlayer</groupId>
         <artifactId>java-client-parent</artifactId>
-        <version>0.1.3</version>
+        <version>0.1.4</version>
     </parent>
     <artifactId>hiddenlayer-sdk</artifactId>
     <packaging>jar</packaging>
     <name>Java Hiddenlayer SDK</name>
```
Comment (wgillett) [inline sdk/pom.xml:12]: nit: HiddenLayer Java SDK

**Review by adrummond-hl — DISMISSED** [review]

**Review by wgillett — DISMISSED** [review]
LGTM with a nit

File: sdk/pom.xml [hunk 2]
```diff
@@ -5,12 +5,33 @@
...
         <groupId>com.hiddenlayer</groupId>
         <artifactId>java-client-parent</artifactId>
-        <version>0.1.3</version>
+        <version>0.1.4</version>
     </parent>
     <artifactId>hiddenlayer-sdk</artifactId>
     <packaging>jar</packaging>
-    <name>Java Hiddenlayer SDK</name>
+    <name>Hiddenlayer Java SDK</name>
     <description>Java SDK for Hiddenlayer</description>
```
Comment (wgillett) [inline sdk/pom.xml]: Hiddenlayer -> HiddenLayer here and above

**Review by wgillett — APPROVED** [review]
LGTM

**Pull request merged**